.github/workflows/deploy.yml

```diff
@@ -0,0 +1,48 @@
+name: Deploy
+on:
+  #push:
+  #  branches: [ main ]
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Environment"
+        required: true
+        default: "enable"
+        type: choice
+        options:
+        - nightly
+        - stable
```

Comment:
This is unused for now?

Reply:
This option it' s being used and provides you with the selection of the environment you want to deploy:
nightly and stable